tests/testthat/test-data_source.R

```diff
@@ -0,0 +1,3 @@
+test_that("function returns static data as list provided by thepackage", {
+  expect_equal(class(agg_data()), "list")
```

Comment:
`agg_data()` will by default return `qmongr::aggr_data`, so maybe change the test to this?:
```
expect_equal(agg_data(), aggr_data)
```